otter/test/models/test_cass_models.py

```diff
@@ -3745,30 +3766,43 @@ def test_get_servers_diff_last_update(self):
         """
         dt_earlier = datetime(2010, 10, 15, 10, 0, 0)
         self._test_get_servers(
-            [{"server_blob": '{"a": "b"}', "last_update": dt_earlier},
-             {"server_blob": '{"d": "e"}', "last_update": dt_earlier},
-             {"server_blob": '{"c": "f"}', "last_update": self.dt}],
+            False,
+            [{"server_blob": '{"a": "b"}', "last_update": dt_earlier,
+              "server_as_active": False},
+             {"server_blob": '{"d": "e"}', "last_update": dt_earlier,
+              "server_as_active": False},
+             {"server_blob": '{"c": "f"}', "last_update": self.dt,
+              "server_as_active": False}],
```

Comment:
Actually sorry I didn't notice before either - although maybe I am just misunderstanding this now.  Should it fetch the latest cache, not the earliest cache?